tests/integration_tests/utils.py

```diff
@@ -223,7 +223,7 @@ def audio_feature(folder, **kwargs):
             {"filter_size": 400, "pool_size": 16, "num_filters": 32, "regularize": "false"},
             {"filter_size": 40, "pool_size": 10, "num_filters": 64, "regularize": "false"},
         ],
-        "fc_size": 256,
+        "output_size": 256,
```

Comment:
these values is likely too big, let's decrease it for faster tests

Reply:
Done